debian/wolkgatewaymodule-modbus.service

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2020 WolkAbout Technology s.r.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+[Unit]
+ Description=WolkAbout Gateway Module for Modbus devices
+ After=multi-user.target
+
+[Service]
+ Type=idle
+ WorkingDirectory=/usr/bin/
+ ExecStart=/usr/bin/modbusModule /etc/modbusModule/moduleConfiguration.json /etc/modbusModule/devicesConfiguration.json

```

Comment:
when I perform make install duplicate of config files appears in /etc/modbusModule with extensiton **.dpkg-new**, Why?

Reply:
First, this file that you have written the comment on has nothing to do with /etc/modbusModule folder.

.dpkg-new files appear as a failure of dpkg to rename the new config files. It will place new files next to the old ones, and try to rename over the old ones.

https://raphaelhertzog.com/2011/07/18/deciphering-one-of-dpkg-weirdest-errors-unable-to-open-pathtofoo-dpkg-new/

Check if your service isn't running before it tries to overwrite those files.